module/zfs/txg.c

```diff
@@ -517,7 +517,8 @@ txg_sync_thread(void *arg)
 		clock_t timeout = zfs_txg_timeout * hz;
 		clock_t timer;
 		uint64_t txg;
-		txg_stat_t *ts;
+		uint64_t dirty_min_bytes =
+		    zfs_dirty_data_max * zfs_dirty_data_sync_percent / 100;
```

Comment:
I'd rather not do a decimal integer division (/100) every pass through this loop because I don't think compilers will optimize this.

Can we place dirty_min_bytes elsewhere and update the assignment when the
values change?

Reply:
I agree with the sentiment, but I'd prefer to leave that change for a follow up PR.

On a related note, there may be significant value in going through the entire code base and replacing all `/` operations with the optimized `div64_*64_*` macros.  To even make the division operator work in the Linux kernel context we needed to implement our own `__udiv*` functions, see `spl-generic.c`.